cmake/functions.cmake

```diff
@@ -79,6 +79,9 @@ function(generate_build_flags)
     set(BUILD_INTEL_ASM ${WOLFSSL_INTEL_ASM} PARENT_SCOPE)
     set(BUILD_AFALG ${WOLFSSL_AFALG} PARENT_SCOPE)
     set(BUILD_DEVCRYPTO ${WOLFSSL_DEVCRYPTO} PARENT_SCOPE)
+    if(HAVE_ARIA OR WOLFSSL_USER_SETTINGS)
```

Comment:
Have you actually tried it with our CMake? Is this really all that was required? If not please add a note about it.